package.json

```diff
@@ -6,7 +6,8 @@
   "dependencies": {
     "@sailshq/lodash": "^3.10.3",
     "async": "^1.5.2",
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
+    "node-fetch": "^2.6.5"
```

Comment:
This is as far into the future as I want to push it because A2 still supports some older node versions and the latest node-fetch (3.x) supports ES modules only.